content/en/policies/other/exclude_namespaces_dynamically.md

```diff
@@ -45,7 +45,7 @@ spec:
     preconditions:
       all:
       - key: "{{request.object.metadata.namespace}}"
-        operator: NotIn
+        operator: AnyNotIn
```

Comment:
If you change this operator to `AnyNotIn` you change minimum version compatibility to 1.6.0, so we must add the appropriate annotation with value. There are many that do need updating, and I consciously avoided this until the next minor version to give users a chance to catch up.

Reply:
Added comment in: https://github.com/kyverno/policies/pull/284